app/javascript/tests/patient/close_contact/CloseContact.test.js

```diff
@@ -30,70 +30,186 @@ describe('CloseContact', () => {
     const alreadyExistingCCWrapper = getShallowWrapper(mockCloseContact2);
-    expect(alreadyExistingCCWrapper.find(Button).length).toEqual(3);
-    expect(alreadyExistingCCWrapper.find(Button).at(0).text()).toContain('Edit');
-    expect(alreadyExistingCCWrapper.find(Button).at(1).text()).toContain('Contact Attempt');
-    expect(alreadyExistingCCWrapper.find(Button).at(2).text()).toContain('View Record');
-    expect(alreadyExistingCCWrapper.state('showModal')).toBeFalsy();
+    expect(alreadyExistingCCWrapper.find(Button).length).toEqual(4);
+    expect(
+      alreadyExistingCCWrapper
+        .find(Button)
+        .at(0)
+        .text()
+    ).toContain('Edit');
+    expect(
+      alreadyExistingCCWrapper
+        .find(Button)
+        .at(1)
+        .text()
+    ).toContain('Contact Attempt');
+    expect(
+      alreadyExistingCCWrapper
+        .find(Button)
+        .at(2)
+        .text()
+    ).toContain('Delete');
+    expect(
+      alreadyExistingCCWrapper
+        .find(Button)
+        .at(3)
+        .text()
+    ).toContain('View Record');
+
+    expect(alreadyExistingCCWrapper.state('showCloseContactModal')).toBeFalsy();
     expect(alreadyExistingCCWrapper.state('first_name')).toEqual(mockCloseContact2.first_name || '');
     expect(alreadyExistingCCWrapper.state('last_name')).toEqual(mockCloseContact2.last_name || '');
     expect(alreadyExistingCCWrapper.state('primary_telephone')).toEqual(mockCloseContact2.primary_telephone || '');
     expect(alreadyExistingCCWrapper.state('email')).toEqual(mockCloseContact2.email || '');
-    expect(alreadyExistingCCWrapper.state('last_date_of_exposure')).toEqual(mockCloseContact2.last_date_of_exposure || null);
-    expect(alreadyExistingCCWrapper.state('assigned_user')).toEqual(mockCloseContact2.assigned_user || null);
+    expect(alreadyExistingCCWrapper.state('cc_last_date_of_exposure')).toEqual(mockCloseContact2.last_date_of_exposure || null);
+    expect(alreadyExistingCCWrapper.state('cc_assigned_user')).toEqual(mockCloseContact2.assigned_user || null);
     expect(alreadyExistingCCWrapper.state('notes')).toEqual(mockCloseContact2.notes || '');
     expect(alreadyExistingCCWrapper.state('contact_attempts')).toEqual(mockCloseContact2.contact_attempts || 0);
   });
 
   it('Properly renders all main components for already existing unenrolled close contact', () => {
     const alreadyExistingCCWrapper = getShallowWrapper(mockCloseContact3);
-    expect(alreadyExistingCCWrapper.find(Button).length).toEqual(3);
-    expect(alreadyExistingCCWrapper.find(Button).at(0).text()).toContain('Edit');
-    expect(alreadyExistingCCWrapper.find(Button).at(1).text()).toContain('Contact Attempt');
-    expect(alreadyExistingCCWrapper.find(Button).at(2).text()).toContain('Enroll');
-    expect(alreadyExistingCCWrapper.state('showModal')).toBeFalsy();
+    expect(alreadyExistingCCWrapper.find(Button).length).toEqual(4);
+    expect(
```

Comment:
I think this PR is good to go after a rebase

Reply:
make sure to update `prettier` with `yarn` (to v2.x) and then run:
```
./node_modules/.bin/pretty-quick --pattern "app/javascript/*(tests|components)/**/*.*(js|jsx)"
```
afterwards because rebasing doesn't trigger `prettier`

Reply:
there are linting errors so this still needs to be done ^

Reply:
@DPC15038, why aren't these showing up in the CI?

Reply:
^ might be because this branch isn't rebased?

Reply:
@timwongj I don't think so, I saw something on my vaccine delete PR sneak through as well